website/content/guides/1.7-template-upgrade.mdx

```diff
@@ -73,10 +94,12 @@ build {
 }
 ```
 
-This template uses the `packer-provisioner-comment` plugin and, until Packer v1.7.0, the binaries were installed manually.
+This template uses the example [packer-provisioner-comment](https://github.com/sylviamoss/packer-plugin-comment) plugin.
```

Comment:
I thought it might be instructive to actually link the repo so users could see it in a less abstract way.